hack/e2e.sh

```diff
@@ -44,6 +44,20 @@ done
 # Allow default serviceaccount to use CI pull secret
 oc secrets link default ci-pull-secret --for=pull
 
+# Reconfigure monitoring operator to support user workloads
+# https://docs.openshift.com/container-platform/4.3/monitoring/monitoring-your-own-services.html
+cat > /tmp/config.yaml << EOF
+techPreviewUserWorkload:
+  enabled: true
+EOF
+oc -n openshift-monitoring create configmap cluster-monitoring-config --from-file=/tmp/config.yaml
```

Comment:
To avoid both the `/tmp` file and [Bash gymnastics][1], how about:

```sh
 oc -n openshift-monitoring create configmap cluster-monitoring-config '--from-literal=config.yaml={"techPreviewUserWorkload": {"enabled": true}}'
```

?

[1]: https://github.com/openshift/cincinnati/pull/248#discussion_r390982511